transaction/examples/metadata/metadata.rtm

```diff
@@ -8,11 +8,96 @@ SET_METADATA
     "k"
     Enum(0u8, Enum(0u8, "v"));
 
+# Set String Metadata on Resource
 SET_METADATA
     Address("resource_sim1q2ym536cwvvf3cy9p777t4qjczqwf79hagp3wn93srvsgvqtwe")
     "k"
     Enum(0u8, Enum(0u8, "v"));
 
+# Set Boolean Metadata on Resource
+SET_METADATA
+    Address("resource_sim1q2ym536cwvvf3cy9p777t4qjczqwf79hagp3wn93srvsgvqtwe")
+    "k"
```

Comment:
Could we - eg - use "field_name" instead of "k"? (in accordance with https://radixdlt.atlassian.net/wiki/spaces/DevEcosystem/pages/2958000129/Metadata+Standards+for+Wallet )